docs/release_notes.rst

```diff
@@ -37,9 +52,77 @@ modular metadata management system that uses `Pydantic
 validate the metadata schema and export to a variety of formats to support data
```

Comment:
github won't let me comment above this line... grr. but this is actually for the bullet above that starts with 

> * The new :mod:`pudl.load.sqlite` and :mod:`pudl.load.parquet` modules contain

The end of this bullet seem out of date bc #1211 is DONE.

Reply:
Okay, updated!